### PR TITLE
Note on SIOPv2 and rework todo

### PR DESCRIPTION
**Pull request by matgnt**

### CODE REVIEW

File: specifications/M1/identity.protocol.base.md [hunk 1]
```diff
@@ -54,13 +54,15 @@ be resolvable via a `DID`.
 
 # 4. Self-Issued ID Tokens
 
-A Self-Issued ID Token is defined in
+A Self-Issued ID Token is defined as in
```
Comment (jimmarino) [inline specifications/M1/identity.protocol.base.md]: "is defined in" is correct

File: specifications/M1/identity.protocol.base.md [hunk 2]
```diff
@@ -54,13 +54,15 @@ be resolvable via a `DID`.
...
 the [Self-Issued OpenID Provider v2 specification](https://openid.net/specs/openid-connect-self-issued-v2-1_0.html#section-1.1) :
 
 > In the Self-Issued OP case, the ID Token is self-signed with a private key under the user's control, identified by the
-> sub-claim.
+> *sub* claim.
 
-A client may obtain a Self-Issued ID Token using a variety or OAuth grant types. If the OAuth 2.0 Client Credential Grant
+> NOTE: This speicifcation does NOT implement or require a full implemenation of the referenced SIOPv2 specification
```
Comment (jimmarino) [inline specifications/M1/identity.protocol.base.md]:
```suggestion
> NOTE: This specification does NOT require a complete implementation of the SIOPv2 specification
```

**Review by jimmarino — CHANGES_REQUESTED** [review]
A few minor changes, then I am ok with this

File: specifications/M1/identity.protocol.base.md (hunk 1 above)
Comment (matgnt) [reply specifications/M1/identity.protocol.base.md]: reverted

**Review by jimmarino — APPROVED** [review]

**Comment by matgnt** [comment]
I did rebase and cleanup the changes.

**Pull request merged**